src/main/java/views/ResultView.java

```diff
@@ -1,8 +1,76 @@
 package views;
 
+import data.Messages;
+import domain.Lotto;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class ResultView {
 
-    public static void println(int num){
+    private static final Map<Long, BigDecimal> prizeMap = new HashMap<>() {{
+        put(3L, BigDecimal.valueOf(5000));
+        put(4L, BigDecimal.valueOf(50000));
+        put(5L, BigDecimal.valueOf(1500000));
+        put(6L, BigDecimal.valueOf(2000000000));
```

Comment:
```suggestion
        put(6L, BigDecimal.valueOf(2_000_000_000));
```
숫자의 경우 적절하게 `_`를 활용해주면 가독성있는 숫자를 표현해줄 수 있습니다 😄